src/app/lib/config/services/loadableConfig.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This config is used by `createLoadableContext`
+ * to dynamically load each service's config.
+ */
+const loadableConfig = {
```

Comment:
Is it possible to remove the duplication here, or is this `webpackChunkName` comment blocking it?

Something like:

```
import arrayOfServiceNames from 'somewhere';

const loadableConfig = arrayOfServiceNames.map(service => ({
  // methods
}));
```

Reply:
Yeah, AFAIK theres no way of dynamically building the comment sadly :(